sdk/identity/azure-identity-v2/src/main/java/com.azure.identity.v2/AzureAuthorityHosts.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.v2;
+
+/**
+ * <p>Defines fields exposing the well known authority hosts for the Azure Public Cloud and sovereign clouds.</p>
+ *
+ * <p>Users looking to target a specific cloud instance of Azure, can pass an instance of Azure Authority Hosts
+ * as a parameter to the {@link EntraIdCredentialBuilderBase#authorityHost(String)}. The authentication requests
+ * will be routed accordingly to the specified target authority host.</p>
+ *
+ * @see EntraIdCredentialBuilderBase
+ */
+public final class AzureAuthorityHosts {
```

Comment:
I wonder if we can lose this in favor of the impl Xiang wants in Core (which I assume we will bring to core-v2 also). If we end up doing it in core let's make sure we don't repeat the values here and carry them forever.

Reply:
Added a TODO to move it to azure-core-v2, as needed